python/nav/web/navlets/__init__.py

```diff
@@ -266,7 +271,12 @@ def dispatcher(request, navlet_id):
             navlet_id,
             owner,
         )
-        return HttpResponse(status=403)
+        if request.htmx:
+            return _handle_htmx_error_response(
+                request, account_navlet, 'Not authorized to view this widget'
+            )
+        else:
+            return HttpResponse(status=403)
```

Comment:
Above the structure 
```
if request.htmx:
    return ...
return HttpResponse(...)
```
is used and here
```
if request.htmx:
    return ...
else:
    return HttpResponse(...)
```

I am happy with either variation, but it should be consistent